static/app/components/sidebar/index.tsx

```diff
@@ -417,7 +436,11 @@ function Sidebar() {
       <SidebarItem
         {...sidebarItemProps}
         icon={<IconLightning />}
-        label={<GuideAnchor target="performance">{t('Performance')}</GuideAnchor>}
+        label={
+          <GuideAnchor target="performance">
+            {hasNewNav ? truncateString('Performance', 4) : t('Performance')}
```

Comment:
you could just hardcode it to Perf

Reply:
😅 ya tru.
might as well just do that then lol